src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSProject_IVsENCRebuildableProjectCfg.cs

```diff
@@ -16,69 +17,44 @@ public int HasCustomMetadataEmitter(out bool value)
         }
 
         public int StartDebuggingPE()
-        {
-            return _editAndContinueProject.StartDebuggingPE();
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int StopDebuggingPE()
-        {
-            return _editAndContinueProject.StopDebuggingPE();
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetPEidentity(Guid[] pMVID, string[] pbstrPEName)
-        {
-            return _editAndContinueProject.GetPEidentity(pMVID, pbstrPEName);
-        }
+            => 0; // TODO: shouldn't be called
 
         public int EnterBreakStateOnPE(EncInterop.ENC_BREAKSTATE_REASON encBreakReason, ShellInterop.ENC_ACTIVE_STATEMENT[] pActiveStatements, uint cActiveStatements)
-        {
-            return _editAndContinueProject.EnterBreakStateOnPE(encBreakReason, pActiveStatements, cActiveStatements);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetExceptionSpanCount(out uint pcExceptionSpan)
-        {
-            pcExceptionSpan = default;
-            return _editAndContinueProject.GetExceptionSpanCount(out pcExceptionSpan);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetExceptionSpans(uint celt, ShellInterop.ENC_EXCEPTION_SPAN[] rgelt, ref uint pceltFetched)
-        {
-            return _editAndContinueProject.GetExceptionSpans(celt, rgelt, ref pceltFetched);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetCurrentExceptionSpanPosition(uint id, VsTextSpan[] ptsNewPosition)
-        {
-            return _editAndContinueProject.GetCurrentExceptionSpanPosition(id, ptsNewPosition);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetENCBuildState(ShellInterop.ENC_BUILD_STATE[] pENCBuildState)
-        {
-            return _editAndContinueProject.GetENCBuildState(pENCBuildState);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int ExitBreakStateOnPE()
-        {
-            return _editAndContinueProject.ExitBreakStateOnPE();
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetCurrentActiveStatementPosition(uint id, VsTextSpan[] ptsNewPosition)
-        {
-            return _editAndContinueProject.GetCurrentActiveStatementPosition(id, ptsNewPosition);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int EncApplySucceeded(int hrApplyResult)
-        {
-            return _editAndContinueProject.EncApplySucceeded(hrApplyResult);
-        }
+            => throw ExceptionUtilities.Unreachable;
 
         public int GetPEBuildTimeStamp(Microsoft.VisualStudio.OLE.Interop.FILETIME[] pTimeStamp)
         {
             return VSConstants.E_NOTIMPL;
```

Comment:
❔ This one is still called for something?

Reply:
No. 

---
In reply to: [270625491](https://github.com/dotnet/roslyn/pull/34027#discussion_r270625491) [](ancestors = 270625491)